project/npda/templates/partials/patient_table.html

```diff
@@ -44,9 +44,9 @@
               <tr>
                 <th colspan="13" class="px-6 py-3">
                   {% if patient.is_first_valid %}
-                    Patients where all records have been validated: {{ valid_count_in_page }}
+                    Patients where all records have been validated: {{ valid_count_in_page }} of {{submission_valid_count}}
```

Comment:
Perhaps should say "Patients where all records have been validated: (showing X of Y)"

Reply:
I've removed the counts for now since it's not obvious it's showing you errors 51 through 100 etc